src/main/java/com/Main.java

```diff
@@ -105,6 +105,11 @@ public static CanvasGetter setupCanvasGetter(String userId) {
     public static void main(String[] args) throws Exception {
         App app = new App();
 
+        // To prevent heroku from sleeping 
```

Comment:
How does this prevent Heroku from sleeping? Do we have to call this endpoint from Slack?

Reply:
I have a cronjob from http://cron-job.org/ running every 30-60 minutes, so if it reaches this endpoint, with a 200 OK status, it will continue doing so (and keep the application up!)

Reply:
Great.